nav_msgs/Path reference_path
nav_msgs/Path tracked_path
float32[] err_lat
float32[] cross_track